Formula/d2.rb

```diff
@@ -30,7 +30,7 @@ def install
       homebrew-core -> brew: depends
     EOS
 
-    system bin/"d2", test_file
+    system bin/"d2", "test.d2"
```

Comment:
@Porkepix why did this change in the bump?

Reply:
hm strange, we'll look into it. thank you for bumping 🙏

Reply:
It wasn't changed by the bump but by @chenrui333 who did this to fix the broken tests.

Reply:
Fixed this bug in master so next release this can be reverted.

Reply:
https://github.com/terrastruct/d2/pull/979

Reply:
it'll continue to work without reverting anyway right

Reply:
Yup, but I believe the convention is to use `test_file` and not hard code the path. It doesn't matter much though yes.